opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -170,275 +216,6 @@ class SpanKind(enum.Enum):
     CONSUMER = 4
```

Comment:
I wonder if `SpanKind`  should also be in `span.py`

Reply:
interestingly SpanKind is not used in the Span ABC itself.. interestingly they rejoin in the SDK.

I'm indifferent here. I'll look through other comments and make a final call.